importer_jenkins_setup.sh

```diff
@@ -80,7 +85,45 @@ sleep 10
 
 echo "specify db populated"
 
+# setting up test server
+(
```

Comment:
stable_cas-server is configured on a seperate port than the main server pipeline and only fetches and deploys the master branch.